internal/provisioner/plain/README.md

```diff
@@ -235,51 +279,58 @@ spec:
       provisionerClassName: core.rukpak.io/plain
       source:
         image:
-          ref: quay.io/tflannag/bundle:combo-operator-v0.0.2
+          ref: quay.io/operator-framework/combo-bundle:v0.0.2
         type: image
 EOF
 ```
 
-And wait until the newly generated `combo-xzfxv` Bundle is reporting an Unpacked status:
+Once the newly generated `combo-xzfxv` Bundle is reporting an Unpacked status, the BundleInstance `combo`
+resource should now point to the new Bundle version. The combo-operator deployment
+in the combo namespace should also be healthy and contain a new container image:
 
-```console
+```text
 $ kubectl get bundles -l app=combo
 NAME           TYPE    PHASE      AGE
-combo-9njsj     image  Unpacked   30s
+combo-9njsj    image   Unpacked   30s
 combo-xzfxv    image   Unpacked   10s
-```
 
-And verify that the BundleInstance `combo` resource now points to the new Bundle version:
-
-```console
 $ kubectl get bundleinstance combo
 NAME    INSTALLED BUNDLE   INSTALL STATE           AGE
 combo   combo-xzfxv        InstallationSucceeded   10s
-```
-
-And check that the combo-operator deployment in the combo namespace is healthy and contains a new container image:
 
-```console
 $ kubectl -n combo get deployment
 NAME             READY   UP-TO-DATE   AVAILABLE   AGE
 combo-operator   1/1     1            1           10s
 
 $ kubectl -n combo get deployments.apps combo-operator -o yaml | grep 'image:' | xargs
-image: quay.io/tflannag/combo:v0.0.2
+image: quay.io/operator-framework/combo-operator:v0.0.2
 ```
 
-Notice that the container image has changed since we had first installed the combo operator.
+Notice that the container image has changed to `v0.0.2` since we first installed the combo operator.
 
-### Deleting the Combo Operator
+### Deleting the Combo Operator and Local Kind Cluster
 
-To cleanup from the installation, simply remove the BundleInstance from the cluster. This will remove all references
+To clean up from the installation, simply remove the `BundleInstance` from the cluster. This will remove all references
```

Comment:
nit: do we introduce inconsistency using `BundleInstance` vs. BundleInstance?

Reply:
I'll have to check but I thought I made that change to make it consistent in that section of the doc.

Reply:
It looks like it's kind of inconsistent throughout the doc. Is there a preference for one way over the other?
cc: @exdx

Reply:
I'm fine with whatever as long as we're consistent. I think we typically use BundleInstance without the code brackets throughout the rest of the documentation set, so it's also possible we have this issue outside of this doc as well.